integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/putmessage/StrategyFactory.java

```diff
@@ -33,17 +31,17 @@ public StrategyFactory(MessageSender messageSender, ServiceRegistryLookup servic
         }
 
         factories = new HashMap<>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Convert this Map to an EnumMap. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1640)